docs/scripts/filter.js

```diff
@@ -83,3 +93,42 @@ function sessionParser(){
     filtersParsed = JSON.parse(filtersParsed)
     return filtersParsed
 }
+
+function signUp(){
+    var email = document.getElementById("email-sign-up").value;
+    var username = document.getElementById("username-sign-up").value;
+    var password = document.getElementById("password-sign-up").value;
+    var passwordCon = document.getElementById("password-confirm-sign-up").value;
+    if (password != passwordCon){
+        alert("Passwords don't match");
```

Comment:
Good job consider showing an alert on the front end instead like via UI instead of a pop up that freezes the page. Maybe in a future ticket